doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -291,6 +310,6 @@ The nRF70 Series driver can be used in the following profiles (not an exhaustive
        ``UDP-RX: 12.7 Mbps``
 
 .. note::
-   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002 DK. The results represent the best throughput, averaged over three iterations, and were obtained with a good RSSI signal in a clean environment.
+   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002DK. The results represent the best throughput, averaged over three iterations, and were obtained with a good RSSI signal in a clean environment.
 
    The above configuration values can be passed using standard ways such as CMake arguments to west build (`Zephyr One Time Arguments`_ or `Zephyr Permanent Arguments`_), or by adding them in an :file:`overlay` file and passing to west build as CMake argument ``EXTRA_CONF_FILE``. See `Zephyr Application Configuration`_ for more details.
```

Comment:
```suggestion
   The above configuration values can be passed using standard ways such as CMake arguments to west build (`Zephyr One Time Arguments`_ or `Zephyr Permanent Arguments`_), or by adding them in an :file:`overlay` file and passing to west build as CMake argument ``EXTRA_CONF_FILE``.
   See `Zephyr Application Configuration`_ for more details.
```

Reply:
I guess this one is missed.

Reply:
I did this, but lost in rebase, its fixed now.